src/plugins/tables/tables.js

```diff
@@ -84,6 +84,29 @@ var componentName = "wb-tables",
 						dataTableExt = $.fn.dataTableExt;
 
 					/*
+					* Add printing support
+					*/
+					if ( $elm.hasClass( "wb-tables-print" ) ) {
+						var $btn = $( "<button class='btn btn-default mrgn-bttm-md' id='" + elmId + "_datatable-print'>Print Table</button>" );
+						$elm.before( $btn );
+						$btn.click( function() {
+							var css = "<link href='" + wb[ "/" ].replace( "js", "css" ) + "/theme" + wb.getMode() + ".css' rel='stylesheet' />", table = $( "<div>" ).append( $elm.clone() ).html(), html = css + table, printWin = window.open( "about:blank", "_blank" );
```

Comment:
When running grunt test it didn't like it that way, but I'll try again maybe I had to much whitespace.